executor/aggregate.go

```diff
@@ -344,6 +349,15 @@ func (e *HashAggExec) initForUnparallelExec() {
 	}
 }
 
+func closeBaseExecutor(b *baseExecutor) {
+	if r := recover(); r != nil {
+		if str, ok := r.(string); ok && strings.Contains(str, memory.PanicMemoryExceed) {
+			terror.Log(b.Close())
+			panic(r)
```

Comment:
panic is recovered here if it is not caused by OOM, I think the logic is different from the original.

Reply:
Good point! I will change here...

Reply:
done